frontend/admin/src/js/components/classification/CreateClassification.tsx

```diff
@@ -6,6 +6,7 @@ import { useIntl } from "react-intl";
 import { Input, Select, Submit } from "@common/components/form";
 import { toast } from "@common/components/Toast";
 import { errorMessages } from "@common/messages";
+import Heading from "@common/components/Heading/Heading";
```

Comment:
Shorten import
```suggestion
import Heading from "@common/components/Heading";
```

I think this can be done throughout the files since the index.ts file exports the component, no?

Reply:
There was [a slack discussion](https://talent-cloud.slack.com/archives/C0259G6KXJ6/p1666099717196299) on the use of `index.ts` and it perhaps being a bad practice. I've been trying to avoid it now where possible but if this is not the place to do that, I can update them.

Reply:
I see, ok. Is there an issue on the board already for this? If not, worth converting the following into an issue:

- [ ] remove index.ts files and update the imports accordingly

If not, worth creating. Add it to the [debt](https://github.com/GCTC-NTGC/gc-digital-talent/labels/debt) pile with this [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/1877)!

Reply:
I'm not sure @patcon did we ever create an issue for the import paths?